### ISSUE TITLE
Showing flux fields based on conditions doesn't work anymore

### ISSUE THREAD

**Issue by kledo-34** [FluidTYPO3/flux#1857]
Hey there, before typo3 10 and latest flux version the following was a working piece of code:

`<flux:field.select name="Auswahl" maxItems="1" label="Art" requestUpdate="true" default="none" items="{	
						1:{0:'option 1',1:'first'},
						2:{0:option 2',1:'second'}
						}"/>	`

`<f:if condition="{Auswahl} == 'first'">
	<flux:field.input ....
</f:if>`

`<f:if condition="{Auswahl} == 'second'">
	<flux:field.input ....
</f:if>`

Basically it's a select field and depending on the selection different flux fields are shown. This doesn't work anymore, the fields inside the f:if are never shown. I don't know if it has to do with typo3's f:if viewhelpers or with flux.
Could you please tell me if this is going to be fixed or if this is an obsolete way of implementing this, if so is there a newer way to archieve this.
Thanks a lot.

**Comment by kledo-34** [comment]
I solved it by switching to the displayCond attribute. Furthermore I had to pass parameters="{_all}" to partials to access the values entered in the partials flux fields itself which was not necessary in versions before. Anyways solved so far.
